src/components/Auth/Landing.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, memo, useCallback, useState } from 'react';
+
+import { LandingHeader, LandingBody, LandingFooter } from 'components/Auth';
+import LoginModal from './LoginModal';
+import ResetModal from './ResetModal';
+import LinkSentModal from './LinkSentModal';
+import NewPasswordModal from './NewPasswordModal';
+import PasswordResetModal from './PasswordResetModal';
+import SignupModal from './SignupModal';
+import useStyles from './styles';
+
+const Landing: FC = () => {
+  const styles = useStyles();
+
+  const [openLogin, setOpenLogin] = useState<boolean>(false);
+  const [openSignup, setOpenSignup] = useState<boolean>(false);
+  const [openReset, setOpenReset] = useState<boolean>(false);
+  const [openLinkSent, setOpenLinkSent] = useState<boolean>(false);
+  const [openNewPassword, setOpenNewPassword] = useState<boolean>(false);
+  const [openPasswordReset, setOpenPasswordReset] = useState<boolean>(false);
+
+  const switchToLogin = useCallback((): void => {
+    setOpenSignup(false);
+    setOpenReset(false);
+    setOpenLogin(true);
+    setOpenPasswordReset(false);
+  }, []);
+
+  const switchToReset = useCallback((): void => {
+    setOpenSignup(false);
+    setOpenLogin(false);
+    setOpenReset(true);
+  }, []);
+
+  const switchToSignup = useCallback((): void => {
+    setOpenLogin(false);
+    setOpenReset(false);
+    setOpenSignup(true);
+  }, []);
+
+  const switchToLinkSent = useCallback((): void => {
+    setOpenReset(false);
+    setOpenLinkSent(true);
+  }, []);
+
+  const switchToPasswordReset = useCallback((): void => {
+    setOpenPasswordReset(true);
+    setOpenNewPassword(false);
+  }, []);
+
+  // TODO: remove this - just setup to show the new modal
+  const removeThis = useCallback((): void => {
+    setOpenLinkSent(false);
+    setOpenLogin(false);
+    setOpenReset(false);
+    setOpenSignup(false);
+    setOpenPasswordReset(false);
+    setOpenNewPassword(true);
+  }, []);
+
+  return (
+    <div className={styles.landing}>
+      <LandingHeader
+        openLogin={(): void => setOpenLogin(true)}
```

Comment:
There are a bunch of wdyr issues I think they're all related to using arrow functions in the props instead of using callbacks